### ISSUE TITLE
-gui not working on Windows

### ISSUE THREAD

**Issue by jjjesus** [plantuml/plantuml#163]
Invoking with:

java -jar plantuml.jar -gui
No diagram found

**Comment by arnaudroques** [comment]
Many thanks for the report.
This is a severe regression, that is fixed in last beta http://beta.plantuml.net/plantuml.jar
Tell us if it's not working for you!

**Comment by jjjesus** [comment]
Thanks, I got the last Beta and it's working fine, thanks.

**Comment by Aigeruth** [comment]
I experience the same issue:

```shell
➜ java -jar /usr/local/Cellar/plantuml/1.2018.14/libexec/plantuml.jar --gui
No diagram found
➜ echo $?
100
```

I've tried the beta version as well and it works correctly.

**Comment by arnaudroques** [comment]
Thanks for the feedback. This should be solved with version 1.2019.0

**Comment by Aigeruth** [comment]
It is resolved for me. IMHO, this issue can be closed.
